Tests/PackageLoadingTests/PD_5_7_LoadingTests .swift

```diff
@@ -22,6 +22,33 @@ class PackageDescription5_7LoadingTests: PackageDescriptionLoadingTests {
         .v5_7
     }
 
+    func testRegistryDependencies() throws {
```

Comment:
Does this test just loads/parses the manifest? Does it actually resolve deps?

Reply:
only parses!